package.json

```diff
@@ -33,6 +33,12 @@
       ]
```

Comment:
Nit: change "name" in package.json to `live-session-vscode-extension` for consistency

Reply:
This will also impact the generated VSCode extension file name, so the README will have to be updated